README.md

```diff
@@ -6,6 +6,87 @@ This project provides:
 
-[Akka Discovery]: https://doc.akka.io/docs/akka/2.5/discovery/index.html
+## Configuration for Lagom Java
+
+For `sbt` users, add the dependencies to your `build.sbt` file:
+
+```sbt
+lazy val `hello-impl` = (project in file("hello-impl"))
+  .enablePlugins(LagomJava)
+  .settings(
+    libraryDependencies ++= Seq(
+      "com.lightbend.lagom" %% "lagom-javadsl-akka-discovery-service-locator" % "0.0.12",
+
+      // for example, include Akka Discovery Kubernetes implementation
+      // see https://doc.akka.io/docs/akka/current/discovery/index.html
+      "com.lightbend.akka.discovery" %% "akka-discovery-kubernetes-api" % "1.0.0-RC2"
+    )
+  )
+```
+
+For `maven` users, add the dependencies to your pom.xml
+
+```xml
+<dependency>
+    <groupId>com.lightbend.lagom</groupId>
+    <artifactId>lagom-javadsl-akka-discovery-service-locator_2.12</artifactId>
+    <version>0.0.12</version>
+</dependency>
+
+<!-- 
+  for example, include Akka Discovery Kubernetes implementation
+  see https://doc.akka.io/docs/akka/current/discovery/index.html
+-->
+<dependency>
+    <groupId>com.lightbend.akka.discovery</groupId>
+    <artifactId>akka-discovery-kubernetes-api_2.12</artifactId>
+    <version>1.0.0-RC2</version>
+</dependency>
+```
+
+The Guice module [AkkaDiscoveryServiceLocatorModule](https://github.com/lagom/lagom-akka-discovery-service-locator/blob/master/service-locator/javadsl/src/main/java/com/lightbend/lagom/javadsl/akka/discovery/AkkaDiscoveryServiceLocatorModule.java) will be added by default to your project, but will only wire in the [AkkaDiscoveryServiceLocator](https://github.com/lagom/lagom-akka-discovery-service-locator/blob/master/service-locator/javadsl/src/main/java/com/lightbend/lagom/javadsl/akka/discovery/AkkaDiscoveryServiceLocator.java) when running in production mode. 
+
+In development, your Lagom application will keep using the Lagom's dev-mode `ServiceLocator`. 
+
+Next, you will need to configure Akka Discovery in your `application.conf` file. Consult [Akka Discovery] documentation for further instructions.
+
+
+## Configuration for Lagom Scala
+
+Add the dependencies to your `build.sbt` file:
+
+```sbt
+lazy val `hello-impl` = (project in file("hello-impl"))
+  .enablePlugins(LagomScala)
+  .settings(
+    libraryDependencies ++= Seq(
+      "com.lightbend.lagom" %% "lagom-scaladsl-akka-discovery-service-locator" % "0.0.12",
+
+      // for example, include Akka Discovery Kubernetes implementation
+      // see https://doc.akka.io/docs/akka/current/discovery/index.html
+      "com.lightbend.akka.discovery" %% "akka-discovery-kubernetes-api" % "1.0.0-RC2"
+    )
+  )
+```
+
+Wire [AkkaDiscoveryComponents](https://github.com/lagom/lagom-akka-discovery-service-locator/blob/master/service-locator/scaladsl/src/main/scala/com/lightbend/lagom/scaladsl/akka/discovery/AkkaDiscoveryServiceLocator.scala#L45) in your `ApplicationLoader`:
```

Comment:
Not nice to link with a line number. I will move `AkkaDiscoveryComponents` to its own file on another PR so that problem goes away.